model/Hardware/Properties/buyer.md

```diff
@@ -0,0 +1,17 @@
+SPDX-License-Identifier: Community-Spec-1.0
+
+# buyer
+
+## Summary
+
+A buyer is an individual, organization, or entity that has purchases goods, services, or assets.
```

Comment:
Because the definition also include the purchasing of services, this has a potential to be shared by other profiles (SaaS?) as well, apart from Hardware.

We may able to have this outside of the Hardware profile (Core?).